yarn_api_client/hadoop_conf.py

```diff
@@ -62,34 +57,20 @@ def get_resource_manager_host_port():
 def get_jobhistory_host_port():
     config_path = os.path.join(CONF_DIR, 'mapred-site.xml')
     prop_name = 'mapreduce.jobhistory.webapp.address'
-    value = parse(config_path, prop_name)
-    if value is not None:
-        host, _, port = value.partition(':')
-        return host, port
-    else:
-        return None
+    return parse(config_path, prop_name)
 
 
 def get_nodemanager_host_port():
```

Comment:
Ditto - seems like replacing `host_port` with `endpoint` is the right thing to do.

Reply:
Adjusted